.github/workflows/reusable_pre_release.yaml

```diff
@@ -121,7 +121,9 @@ jobs:
 
   build-win-packages:
     name: Create MSI & Upload into GH Release assets
-    if: ${{ inputs.run_build-win-packages }}
+    # related to https://github.com/actions/runner/issues/2205
+    # since a success() is added by default and skipped jobs make success to fail we need to introduce this workaround
+    if: ${{ always() && needs.build-artifacts.result == 'success' && inputs.run_build-win-packages }}
```

Comment:
🤔 Do we need to do the same in the publish job? It also needs `build-artifacts`

Reply:
in the publish step we have already `${{ always() && !cancelled() && !failure() }}`

Therefore it is not needed 😄

Reply:
Also this could become: 
```
always() && && !cancelled() && !failure() && inputs.run_build-win-packages
```